src/main/java/com/saebyeok/saebyeok/domain/Article.java

```diff
@@ -35,6 +35,7 @@
     @CreatedDate
     private LocalDateTime createdDate;
     private Boolean isCommentAllowed;
+    private Boolean isDeleted;
```

Comment:
```suggestion
    @Builder.Default
    private Boolean isDeleted = Boolean.FALSE;
```
이렇게 사용하면 49라인 삭제할 수 있을 것 같아요!
기본값을 지정할 때 Member.java에서도 이렇게 이용했었는데 통일하는게 어떨까요??

Reply:
이거 적용해주실거면 Comment.isDeleted도 같은 형식으로 부탁드려요 ㅎㅎ

Reply:
적용했습니다. 다만 `@Builder.Default` 어노테이션은 없이도 작동해서 뺐어요.
`@Builder.Default`로 잡으면 빌더로 생성할 때만 기본값 false가 들어가더라고요.